app/Http/Controllers/User/SettingController.php

```diff
@@ -65,7 +73,10 @@ public function postSetting()
             'user_id' => $id,
         ];
         $displayPhone = $this->usermeta->getDataSetting($data);
-        $phones = $this->user->find($id)->get();
+        $data = [
+            'id' =>$id,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/SettingController.php</br>**Line**: 77</br>Expected 1 space between double arrow and "$id"; 0 found